CONTRIBUTING.md

```diff
@@ -38,15 +38,26 @@ Pull requests ("PRs")
 - Try to [tidy your history][git-history-rewriting]: combine related commits
   with interactive rebasing, separate monolithic commits, etc.
 
-### Stages: WIP, RFC
+### Stages: WIP, RFC, RDY
 
-Pull requests have two stages: `[WIP]` (Work In Progress) and `[RFC]` (Request
-For Comment).
+Pull requests have three stages: `[WIP]` (Work In Progress), `[RFC]` (Request
+For Comment) and `[RDY]` (Ready).
 
 - Untagged PRs are assumed to be `[RFC]`, i.e. the work is ready for review and
   you would like feedback.
```

Comment:
since a push is needed anyway, let's change this to:

    , i.e. you are requesting a review.